_posts/2021-11-09-logicdrop-automating-quarkus-with-gitlab.adoc

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: "Automating Quarkus with GitLab"
+date: 2021-11-09
+tags: user-story, development-tips, gitlab
+synopsis: "Embracing Quarkus has changed how our developers work and existing processes, especially CI/CD, had to be re-architected to provide a more integrated experience. Every step, from inception, to development through deployment, and finally monitoring, is now orchestrated through GitLab."
```

Comment:
```suggestion
synopsis: "Embracing Quarkus has changed how our developers work and existing processes, especially CI/CD, had to be re-architected to provide a more integrated experience. Every step, from inception, to development through deployment, and finally monitoring, is now orchestrated through GitLab."
```
I think you need to add some context in the synopsis and make it clear it's about your infra at Logicdrop.

Reply:
Rewrote to:

`
Using Quarkus greatly improved our developers productivity and our CI/CD process needed to be redesigned to catch up. In doing so, we wanted a more universal experience and leveraging Quarkus extensions was a perfect fit. Every step of our Quarkus pipelines, from inception, to development through deployment, and finally monitoring, are now orchestrated through GitLab.
`